packages/react-components/src/components/UnsupportedEnvironment.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* @conditional-compile-remove(unsupported-browser) */
+import { Icon, Link, Stack, Text } from '@fluentui/react';
+/* @conditional-compile-remove(unsupported-browser) */
+import { _pxToRem } from '@internal/acs-ui-common';
+import React from 'react';
+/* @conditional-compile-remove(unsupported-browser) */
+import {
+  containerStyles,
+  iconStyles,
+  linkTextStyles,
+  mainTextStyles,
+  secondaryTextStyles
+} from './styles/UnsupportedEnvironment.styles';
+import { UnsupportedBrowserStrings } from './UnsupportedBrowser';
+import { UnsupportedBrowserVersionStrings } from './UnsupportedBrowserVersion';
+
+/**
+ * props for UnsupportedBrowser UI
+ *
+ * @beta
+ */
+export interface UnsupportedEnvironmentProps {
+  /** Handler to perform a action when the help link is actioned */
+  onTroubleshootingClick?: () => void;
+  /** String overrides for the component */
+  strings: UnsupportedBrowserStrings | UnsupportedBrowserVersionStrings;
```

Comment:
`UnupportedBrowserStrings` and `UnsupportedBrowserVersionStrings` are, in fact, the same shape (have the same fields).

Assuming this is a `@private` component (see comment above), you can define a new type

```TypeScript
interface UnsupportedEnvironmentStrings {
  primaryText: string;
  ...
}
```

and use that. When `UnsupportedBrowser` and `UnsupportedBrowserVersion` forward their own `strings` prop, it will _just work_ because the shape of all those interfaces is the same.